services/web/server/src/simcore_service_webserver/catalog/_rpc.py

```diff
@@ -0,0 +1,92 @@
+""" RPC client-side for the RPC server at the payments service
+
+"""
+
+import logging
+
+from aiohttp import web
+from models_library.api_schemas_catalog import CATALOG_RPC_NAMESPACE
+from models_library.api_schemas_catalog.services import DEVServiceGet, ServiceUpdate
+from models_library.products import ProductName
+from models_library.rabbitmq_basic_types import RPCMethodName
+from models_library.rpc_pagination import PageLimitInt, PageRpc
+from models_library.services_types import ServiceKey, ServiceVersion
+from models_library.users import UserID
+from pydantic import NonNegativeInt, parse_obj_as
+from servicelib.logging_utils import log_decorator
+
+from ..rabbitmq import get_rabbitmq_rpc_client
+
+_logger = logging.getLogger(__name__)
+
+
+@log_decorator(_logger, level=logging.DEBUG)
+async def list_services_paginated(  # pylint: disable=too-many-arguments
+    app: web.Application,
+    *,
+    product_name: ProductName,
+    user_id: UserID,
+    limit: PageLimitInt,
+    offset: NonNegativeInt,
+) -> PageRpc[DEVServiceGet]:
+    rpc_client = get_rabbitmq_rpc_client(app)
+
+    result = await rpc_client.request(
+        CATALOG_RPC_NAMESPACE,
+        parse_obj_as(RPCMethodName, "list_services_paginated"),
+        product_name=product_name,
+        user_id=user_id,
+        limit=limit,
+        offset=offset,
+    )
```

Comment:
TIP: we tend to bundle these inside service lib, that way we recuse errors when calling the interface for and RPC endpoint. Please have a look at https://github.com/ITISFoundation/osparc-simcore/blob/d25082c7934cdd4574c94e8d29bc9b6aeb1c01d6/packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/dynamic_scheduler/services.py

Please keep using this pattern. The test location is OK.

Reply:
good point. 
- Added in `packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/catalog`
- Tests this client in catalog service within  `services/catalog/tests/unit/test_api_rpc.py`